src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/UIstrings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "${ModID}:DimensionGate3Pack": {
+    "TEXT": [
+      "Gate 3: Life & Death",
+      "Cards portalled in from other games! Self-inflict damage for potent effects, and heal up what you've lost.",
+      "MichaelMayhem",
+      "Cards featured from the games: NL One Step From Eden NL Tower Tactics Liberation NL Monster Train NL Inscryption NL Storybook Brawl NL NL Check them all out on Steam!"
+    ]
+  },
+  "${ModID}:DimensionGate3PackHat": {
+    "TEXT": [
+      "Something in the Wildwood unsettles my mind, granting me visions of its very Roots.",
+      "Wyldenten Mask"
```

Comment:
Wyldenten is great. Love the big tree demon. Seeing this makes me want to fire up Monster Train and do an Awoken/X run.

Reply:
I've actually never played since they added it!  I just saw it as the best opportunity for a hat.

Reply:
Wyldenten's one of my favorites. Very versatile, one of the champions that can consistently be one of your main sources of damage and/or control, with every upgrade path being viable.